jest.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
+  testMatch: [
+    "**/__tests__/**/*.test.ts?(x)"
```

Comment:
We search for `.test` file extension, which allows us to put some other files inside the __tests__ directory. Example of a such file, excluded from tests but need by tests:
```
src/background/init/migrations/__tests__/helpers.ts
```